src/pre_defined_classes/mathematical_program/hard/q5/LeftRightTest.java

```diff
@@ -0,0 +1,19 @@
+package pre_defined_classes.mathematical_program.hard.q5;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.*;
+
+public class LeftRightTest extends BaseTest {
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("The leftHandSide is -0.7760683270883323"),
```

Comment:
Should use DoubleLiteral here so we can check the math

Reply:
Looking at the distractors they're all compiler issues, so they cannot possibly get the math wrong, so I didn't see the point in checking that.